### DISCUSSION TITLE
Instant update & highlight by default

### DISCUSSION THREAD

**Discussion by MattiaPARRINELLO** [yandeu/five-server#59] [Help / Questions]
Hello, 
There is a way to set by default instant update and highlight to on, like without creating a `fivserver.config.js` for every workspace?
It's a little boring to always create a file, this parameters can be added to `settings.json`?
Thank you
Great Day

PS: Sorry for my English, I'm not fluent

**Comment by steveja42** [comment]
+1
The only reason I installed this extension was for instant update, so when it didn't work I had  to search the issues, find this issue, and setup a config file for what I expected would just work right away because the description says right at the top: "instant updates"
